<launch>
    <node name = "camera_video_publisher" pkg = "camera" type = "camera_video_publisher.py" output="screen"/> 
    <!-- <include file="$(find surge_et_ambula)/launch/humanoid_simul.launch"/> -->
    <node name="ball_detection" pkg="goal_detection" type="ball_detection" output="screen"/>
    <node name="ball_tracker" pkg="goal_detection" type="ball_tracker" output="screen" required="true">
        <!-- <param name="head_point1"  value="$(find goal_detection)/scripts/ball_detection/head_point1.npz"/>
        <param name="head_point2"  value="$(find goal_detection)/scripts/ball_detection/head_point2.npz"/>
        <param name="head_point3"  value="$(find goal_detection)/scripts/ball_detection/head_point3.npz"/>
        <param name="head_point4"  value="$(find goal_detection)/scripts/ball_detection/head_point4.npz"/> -->
    </node>
    <rosparam command="load" file="$(find goal_detection)/config/vision_config_hardware.yaml" />
</launch>
